lib/infra/infra-stack.ts

```diff
@@ -425,6 +425,10 @@ export class InfraStack extends Stack {
       const baseConfig: any = load(readFileSync(`${configFileDir}/multi-node-base-config.yml`, 'utf-8'));
 
       baseConfig['cluster.name'] = `${scope.stackName}-${scope.account}-${scope.region}`;
+
+      // use discovery-ec2 to find manager nodes by querying IMDS
+      baseConfig['discovery.ec2.tag.Name'] = `${scope.stackName}/seedNodeAsg,${scope.stackName}/managerNodeAsg`;
```

Comment:
The same tag name should be associated with ec2 nodes as well, right?
https://github.com/opensearch-project/opensearch-cluster-cdk/blob/main/lib/infra/infra-stack.ts#L195

Reply:
It's already there. Here's what a node gets:

<img width="1201" alt="Screenshot 2023-06-08 at 1 58 09 PM" src="https://github.com/opensearch-project/opensearch-cluster-cdk/assets/542335/9fc6ca1b-cb45-461e-a1d8-a4d73c3811ed">